src/docs/Components/Fields/previews/fields-state-mandatory.preview.html

```diff
@@ -0,0 +1,31 @@
+<div class="example">
+  <div class="mc-fields">
+    <label class="mc-fields__label" for="hover">Label<span class="mc-fields__mandatory">mandatory</span></label>
+    <input type="text" class="mc-text-input is-hover" id="hover" placeholder="Hover" name="example" />
```

Comment:
Remove all the `is-hover` and `is-focus` occurrences in your previews, we will remove them from everywhere in the docs